backend/render_router.py

```
@@ -0,0 +1,179 @@
+
+    The distinction matters to the dashboard: 503 means "Render is unreachable,
+    retry", 500 would mean "this endpoint is broken".
+    """
+    return HTTPException(status_code=503, detail=f"Render MCP unavailable: {exc}")
+
+
+def build_render_router(get_current_user: Callable[..., Any]) -> APIRouter:
+    router = APIRouter(prefix="/api/render", tags=["render"])
+
+    @router.get("/health")
+    async def render_health(user: dict = Depends(get_current_user)) -> dict:
```

Ruff (0.16.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)